ksql-rest-app/src/main/java/io/confluent/ksql/rest/util/EntityUtil.java

```diff
@@ -53,64 +55,34 @@ private EntityUtil() {
     }
 
     return fields.stream()
-        .map(field -> new FieldInfo(field.name(), getSchema(field.schema())))
+        .map(field -> new FieldInfo(field.name(), getSchema(field.type())))
         .collect(Collectors.toList());
   }
 
-  private static SchemaInfo getSchema(final Schema schema) {
-    return SchemaWalker.visit(schema, new Converter());
+  private static SchemaInfo getSchema(final SqlType type) {
+    return SqlTypeWalker.visit(type, new Converter());
```

Comment:
Switch code to use `SqlTypeWalker` to walk the SQL type system, rather than the Connect system... which simplifies the visitor code.